web/src/OverviewTable.tsx

```diff
@@ -163,8 +175,16 @@ export const ResourceTableRow = styled.tr`
   color: ${Color.gray60};
   padding-top: 6px;
   padding-bottom: 6px;
+  padding-left: 4px;
 
-  &.isDisabled {
```

Comment:
yes! in the new designs, there aren't any "top-level" row styles. We implement it as styling on individual cells. (most of the changes are semantic markup changes on the cells rather than on the row)

Reply:
great! thanks for explaining.